src/button/ReactionToggleButton.module.css

```diff
@@ -0,0 +1,66 @@
+.raisedButton > svg {
+  color: var(--cpd-color-icon-on-solid-primary);
+}
+
+.reactionPopupMenu {
+  display: flex;
+}
+
+/* These styles override the Modal styles to ensure our emoji
+   picker appears near the toolbar */
+
+.reactionPopupMenuModal {
+  width: fit-content !important;
+  top: 82vh !important;
+}
+
+@media not ((hover: none) or (pointer: coarse)) {
+  .reactionPopupMenuModal > div > div {
```

Comment:
This is for styling the modal in the drawer case, right? Can the `classNameDrawer` prop help make this less brittle?